Makefile

```diff
@@ -186,5 +186,10 @@ build-docker: ## Build m3db-operator docker image with go binary
 	@echo "+ $@"
 	@./build/build-docker.sh
 
+.PHONY: publish-helm-charts
+publish-helm-charts: ## pushes a new version of the helm chart
+	@echo "+ $@"
+	./helm/hack/package.sh
```

Comment:
Where's this script? I can't find it

Reply:
Sorry pulled a n00b mistake and didn't add it. It's now added.